tests/test_ratio_of_means.py

```diff
@@ -99,10 +94,18 @@ def test_ratio_of_means(self, base_file_name):
                 filters=filter_df,
             )
 
-        # imputed_value is in a seperate column, remove this if otherwise
+        # imputed_value is in a separate column, remove this if otherwise
         actual_output["question"] = actual_output[["question", "imputed_value"]].agg(
             sum, axis=1
         )
+        actual_output = actual_output.rename(
+            columns={
+                "default_link_b_match": "default_backward",
+                "default_link_f_match": "default_forward",
+                "default_link_flag_construction_matches": "default_construction",
+                "flag_construction_matches_pair_count": "flag_match_pair_count",
+            }
+        )
```

Comment:
Why do these need to be renamed?

Reply:
The unit test for ratio of means are taken from the Pyspark versions which use different column names. It might be worth to bulk change these at some point, but there are about 70 files which would need to be updated. Happy to add this onto the tech backlog

Reply:
That makes sense. I agree that it's worth changing them but it's definitely not the top priority